lib/pluginmanager/new.rb

```diff
@@ -0,0 +1,109 @@
+# encoding: utf-8
+require "pluginmanager/command"
+require "erb"
+require "ostruct"
+require "fileutils"
+
+class LogStash::PluginManager::New < LogStash::PluginManager::Command
+
+  parameter "TYPE",   "Type of the plugin {input, codec, filter, output}s", :attribute_name => :type
+  parameter "PLUGIN", "Name of the new plugin", :attribute_name => :plugin_name
+
+  TYPES = [ "input", "filter", "output" ]
+
+  def execute
+    validate_parameters
+
+    source = File.join(File.dirname(__FILE__), "templates", "#{type}-plugin")
+    target = FileUtils.mkdir(File.join(Dir.pwd, full_plugin_name))
+    puts " #{ green("Creating") } #{full_plugin_name}"
+
+    create_scaffold(source, target)
+  end
```

Comment:
I am pretty sure we can have exceptions raised from files manipulation, the primary error that come to my mind is `permission` issues when writing a file. I suspect some people will run this command from under their users instead of root.

Might be good to have some rescue block to give better feedback to the users? WDYT?